overlord/snapstate/backend/link.go

```diff
@@ -191,6 +191,18 @@ func (b Backend) generateWrappers(s *snap.Info, linkCtx LinkContext) error {
 	}
 	cleanupFuncs = append(cleanupFuncs, wrappers.RemoveSnapBinaries)
 
+	// check DaemonStartup settings and do not start services on first
+	// install that are inhibited
+	if linkCtx.FirstInstall {
+		for _, app := range s.Apps {
+			// XXX: strawman
+			// XXX2: make this a proper type instead of a string
+			if app.InstallMode == "daemon-inhibit" {
```

Comment:
my thinking was that to follow "refresh-mode" and what we control do, this should be "disable"